docs/topics/appendix-proficiency-levels.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[proficiency_levels]]
+= Proficiency Levels
+
+Using each mission available on {launcher} requires certain minimum knowledge from you, and provides a certain information to you. For easier understanding, the requirements and concepts are organized in several levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
+
+[discrete]
+== {proficiency-foundational}
+
+The missions rated at foundational proficiency generally require no prior knowledge of the subject matter; they provide general awareness and demonstration of key elements, concepts, and terminology. There are no special requirements on you except those directly mentioned in the description of the given mission.
+
+[discrete]
+== {proficiency-advanced}
+
+When using advanced missions, you must be familiar with the common concepts and terminology of the field the mission is concerned with. You must also be able to perform basic tasks on your own, for example configure services and applications, or administer networks. If configuring a particular service is not in the scope of the mission, only the resulting state is described, not the steps to achieve it.
```

Comment:
@rhoads-zach Revised. Correct me if I am wrong, but shouldn't infinitive forms be used in the second sentence based on the grammatical agreement with 'be able to?' I see no reason to use gerunds.